bench/lib/context.go

```diff
@@ -41,6 +41,9 @@ const (
 
 	// CtxKeyCadenceClient is the name of the context key for the cadence client this cadence worker listens to
 	CtxKeyCadenceClient = ContextKey("ctxKeyCadenceClient")
+
+	// CtxKeyJWT is the name of the context key for the JWT
+	CtxKeyJWT = ContextKey("ctxKeyJWT")
```

Comment:
is `bech/lib` the right place to kave a context key ? I suspect it might be better in the `cli` folder but just placed it there because I found the other ones

Reply:
Yeah should not be here. You can put it int factory.go under cli/